### PR TITLE
Helper param fix

### PR DESCRIPTION
**Pull request by tioback**
This pull request fixes:

* #1282 - size parameter breaking scroll requests with helpers.
* #1283 - maxRetries option with value 0 results in no requests being made.

### CODE REVIEW

File: lib/Helpers.js [hunk 1]
```diff
@@ -114,9 +114,9 @@ class Helpers {
         break
       }
 
-      for (let i = 0; i < maxRetries; i++) {
+      for (let i = 0; i <= maxRetries; i++) {
         response = await this[kClient].scroll({
-          ...querystring,
+          ...this.filterQueryStringForScrolling(querystring),
```
Comment (delvedor) [inline lib/Helpers.js:119]: I would configure the parameters in place, without calling an external function :)
```suggestion
          scroll: querystring.scroll,
          rest_total_hits_as_int: querystring.rest_total_hits_as_int || querystring.restTotalHitsAsInt
```

Comment (tioback) [reply lib/Helpers.js:119]: Sure can, I haven't downloaded the project, yet. This fixes were quick changes made through github's online editor 😊
I'll do the proper changes and commit to this PR.

**Comment by tioback** [comment]
Hey @delvedor, the requested changes were made, so I believe this PR is all set and could be merged.
Please let me know if there's anything else missing.

**Review by delvedor — APPROVED** [review]
LGTM

**Comment by delvedor** [comment]
Great work!

**Pull request merged**